src/components/divider/divider.d.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { FormFieldSpacing } from '../../utils/helpers/options-helper/options-helper';
+
+export interface DividerProps {
+  /** Margin top, this value will be multiplied by the theme spacing constant (8) */
+  marginTop: FormFieldSpacing;
+
+  /** Margin bottom, this value will be multiplied by the theme spacing constant (8) */
+  marginBottom: FormFieldSpacing;
+
+  /** Margin left, any valid css value */
+  marginLeft: string;
```

Comment:
Should `marginLeft` and `marginRight` not have `?` as they don't have default values and aren't required?

Reply:
Good point yes they should!